ci/cron/src/Main.hs

```diff
@@ -243,5 +246,11 @@ main = do
                 Exit.exitSuccess
             else do
                 push_to_s3 docs_folder
-                let latest = Foldable.maximumBy compare_versions gh_resp
-                tell_hubspot latest
+                let gh_latest = Foldable.maximumBy (compare `Function.on` to_v . name) gh_resp
```

Comment:
Do you know that `gh_resp` will be non-empty? Otherwise `maximumBy` will crash.

Reply:
We sort `gh_resp` above in descending order. The maximum will be at the head of this list.

Reply:
I'm fine with the script crashing if it is empty. That would mean something is really wrong and requires manual fixing.

Reply:
> We sort gh_resp above in descending order. The maximum will be at the head of this list.

As written, that sorted list is not saved (and is a list of strings, whereas here I want the full data objects).